e2e-test/src/test/resources/pluginParameters.properties

```diff
@@ -0,0 +1,71 @@
+  { "name": "Salary", "type": "int" }, { "name": "workhours", "type": "int" } ] }
+csvAllDataTypeFile=testdata/file/CSV_DATATYPE_TEST_1.csv
+csvAllDataTypeFileSchema=[{"key":"id","value":"int"},{"key":"name","value":"string"},\
+  {"key":"yearofbirth","value":"int"},{"key":"isdeleted","value":"boolean"},{"key":"email","value":"string"},\
+  {"key":"createddate","value":"string"},{"key":"revenue","value":"string"},{"key":"points","value":"string"},\
+  {"key":"longdatatype","value":"string"},{"key":"doubledatatype","value":"double"},\
+  {"key":"date","value":"string"},{"key":"null","value":"string"},{"key":"BytesData","value":"string"}]
+tsvFile=testdata/file/TSV_TEST.tsv
+tsvFileSchema=[{"key":"testscenarioid","value":"string"},{"key":"testdescription","value":"string"},\
+  {"key":"testconditionid","value":"string"},{"key":"testtype","value":"string"}]
+blobFile=testdata/file/BLOB_TEST.zip
+blobFileSchema=[{"key":"body","value":"bytes"}]
+delimitedFile=testdata/file/DELIMITED_TEST.txt
+delimiter=@
+incorrectDelimiter=#
+delimitedFileSchema=[{"key":"id","value":"int"},{"key":"name","value":"string"},\
+  {"key":"yearofbirth","value":"int"},{"key":"datatype","value":"long"},{"key":"tablename","value":"string"}]
+textFile=testdata/file/TEXT_TEST.txt
+textFileSchema=[{"key":"offset","value":"long"},{"key":"body","value":"string"}]
+outputFieldTestFile=testdata/file/OUTPUTFIELD_TEST.csv
+outputPathField=outputfield
+invalidOutputPathField=invalid
+outputFieldTestFileSchema=[{"key":"id","value":"int"},{"key":"firstname","value":"string"},\
+  {"key":"lastname","value":"string"},{"key":"photo","value":"string"},{"key":"outputfield","value":"string"}]
+overrideSchema=[{"key":"id","value":"float"},{"key":"firstname","value":"string"},\
+  {"key":"lastname","value":"string"},{"key":"photo","value":"string"},{"key":"outputfield","value":"string"}]
+overrideField=id
+invalidOverrideField=invalid
+overrideDataType=float
+readRecursivePath=testdata/file/RECURSIVE_TEST
+readRecursivePathSchema=[{"key":"id","value":"int"},{"key":"name","value":"string"},\
+  {"key":"yearofbirth","value":"int"},{"key":"tablename","value":"string"},{"key":"offset","value":"string"}]
+fileMaxSplitSize=120
+regexPathFilter=.+TEST.csv
+incorrectRegexPathFilter=/tZY
+filePluginOutputFolder=file-plugin-output
+skipHeaderTrue=true
+fileSinkPathSuffix=yyyy-MM-dd-HH-mm-ss
+## FILE-PLUGIN-PROPERTIES-END
+
+## SEND-EMAIL-PROPERTIES-START
+##filePathValue=/Users/muminm/IdeaProjects/27-04-22/hydrator-plugins/e2e-test/target/test-classes/testdata/file/CSV_DATATYPE_TEST_1.csv
+failureCasePathValue=/Users/muminm/IdeaProjects/27-04-22/hydrator-plugins/e2e-test/target/test-classes/testdata/file/CSV_FailureCase_sendEmailAlert.csv
+completion=completion
+success=success
+failure=failure
+emailSender=testsendemailalert@gmail.com
+emailSenderFormat=abc
+emailRecipients=testsendemailalert@gmail.com
+emailRecipientsFormat=invalid,valid
+emailSubject=e2e tests send email alert verification
```

Comment:
1. Add this in TestsetupHooks.java file

@Before(order = 1, value = "@SEND_EMAIL")
public static void setGCSConnectionName() {
  PluginPropertyUtils.addPluginProp("emailSubject", "send-email-"+ UUID.randomUUID());
}

2. add @SEND_EMAIL tag on the scenario 

3.In the scenario change to ->
And Enter input plugin property: "emailSubject" with value: "emailSubject"